package-examples/helloworld-kustomize/README.md

```diff
@@ -0,0 +1,65 @@
+# helloworld-kustomize
+
+This is an example of a kpt package that has a kustomize
+patch in it.
+
+## Steps
+
+1. [Fetch the package](#fetch-the-package)
+2. [View the package contents](#view-the-package-contents)
+3. [Configure functions](#configure-functions)
+4. [Render the declared values](#render-the-declared-values)
+5. [Apply the package](#apply-the-package)
+
+### Fetch the package
+
+Get the example package on to local using `kpt pkg get`
+
+    $ kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/helloworld-kustomize
```

Comment:
Rendering issue:

https://github.com/GoogleContainerTools/kpt/tree/next/package-examples/helloworld-tshirt

Always use block quotes instead of indentation:

```shell
$ kpt ..
```